clients/cobol-lsp-vscode-extension/src/test/suite/testHelper.ts

```diff
@@ -76,6 +77,7 @@ export async function closeActiveEditor() {
     await vscode.commands.executeCommand("undo");
     await sleep(100);
   }
+  await vscode.languages.setTextDocumentLanguage(doc.document, "json");
```

Comment:
Closing the editor did not fire the didClose event. This is a hack for the same

Reply:
Is this issue only for COBOL files?

Reply:
Good point, let me check it for other files as well.

Reply:
I can see a similar behavior for JSON